.github/workflows/ci.yaml

```diff
@@ -139,13 +148,32 @@ jobs:
             .github/container/manifest.yaml
             .github/container/patches
 
+      - name: Create URL ref build args
+        id: source-urlrefs
+        shell: bash -x -e {0}
+        run: |
+          # converts manifest yaml to a json object of {SOFTWARE_NAME: URL#REF, ...}
+          urlrefs=$(
+            cat .github/container/manifest.yaml |\
+            yq -o=json 'to_entries | .[] | select(.value.mode == "git-clone") | {( .key | upcase | sub("-", "_") ): .value.url + "#" + .value.latest_verified_commit}' |\
```

Comment:
IIUC it would just take renaming a few keys (that contain `-`) in `manifest.yaml` and we could consistently use `_` delimiters everywhere?

Reply:
https://github.com/NVIDIA/JAX-Toolbox/blob/330c61b4c0246ee41a165d22e767a059b7226291/.github/container/pip-finalize.sh#L17 is the reason this cannot be done, I think